Assets/Fungus/Scripts/Editor/FlowchartEditor.cs

```diff
@@ -82,14 +82,6 @@ public override void OnInspectorGUI()
                 EditorWindow.GetWindow(typeof(FlowchartWindow), false, "Flowchart");
             }
 
-            GUILayout.Space(10);
-
-            if (GUILayout.Button(new GUIContent("Center View", "Centers the window view at the center of all blocks in the Flowchart")))
```

Comment:
This was actually kind of broken after I modified the zooming and centering. I figured it wasn't relevant unless the flowchart window was actually visible anyway (and now there's a center button in the window), so I got rid of it from the inspector.